README.md

```diff
@@ -33,7 +33,7 @@ In the following table the list of parameters that can be provided to the `pdf_v
 |-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | input                   | The source of the PDF file. Accepts a file path, URL, or binary data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
-| width                   | Width of the PDF viewer in pixels. It defaults to 700 pixels.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| width                   | The width of the PDF viewer defaults to 100% of the layout. Specify in pixels with a numeric value, or as a percentage for relative sizing.                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | height                  | Height of the PDF viewer in pixels. If not provided, the viewer shows the whole content.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
How about having height = 100% by default too?

Reply:
I've tried it and pushed it here https://github.com/lfoppiano/streamlit-pdf-viewer/pull/39/commits/5443edb05d3b235b59f471a593fe2c9b749e7535
But, it's outer height not inner height. The custom javascript returned 0 as value when I retrieve inner height because of iframe problem
ref: https://developer.mozilla.org/en-US/docs/Web/API/Window/outerHeight

Reply:
OK forget about it, we can address the height-related issue in #41

Reply:
So, do you think I should revert it? Because I've already pushed the code which supports outerHeight.